packages/react-components/react-button/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -13,6 +13,7 @@ export const buttonClassNames: SlotClassNames<ButtonSlots> = {
 const iconSpacingVar = '--fui-Button__icon--spacing';
 
 const buttonSpacingSmall = '3px';
+const buttonBorderRadiusSmall = '4px';
```

Comment:
This should use [the token](https://github.com/microsoft/fluentui/blob/master/packages/tokens/src/global/borderRadius.ts#L6) for this value. I believe we used `buttonSpacingSmall` before because there is no `3px` border radius token.

Reply:
Hi @spmonahan, thanks for pointing it out. Yes you're right, there's no 3px border-radius token in there and as per design specs, it must be 4px.
<img width="472" alt="image" src="https://github.com/microsoft/fluentui/assets/28218967/fb0be261-3ba0-4d98-9ed2-a92c02da39cf">

Reply:
Gonna update the code to use value from the token you shared.